protobuf/service/service.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
```

Comment:
maybe we can move this file in `protobuf/api/service.proto` (https://forum.mesg.com/t/protobuf-folder-structure/297)

Reply:
done